cmake/utils.cmake

```diff
@@ -90,3 +90,57 @@ function(add_shaderc_tests)
       COMMAND ${TEST_NAME})
   endforeach()
 endfunction(add_shaderc_tests)
+
+# Finds all transitive static library dependencies of a given target.
+# This will not find libraries that were statically linked that were not
```

Comment:
I'd say "skip" instead of "not find".
I found it harder to read otherwise.
(But this is oh so super minor.)